src/steamship/data/block.py

```diff
@@ -22,12 +22,6 @@ class Block(CamelModel):
     text: str = None
     tags: Optional[List[Tag]] = []
 
-    class CreateRequest(Request):
-        id: str = None
-        file_id: str = None
-        text: str = None
-        tags: Optional[List[Tag.CreateRequest]] = []
-
```

Comment:
Was the removal of `Block.CreateRequest` and `Tag.CreateRequest` something that was necessary for the kind-specific tag classes, or just felt right along the way?

I don't really have a clear opinion / solution in my head, because I find the need to use the `CreateRequest` s VERY annoying, but at the same time, the `File` object and its `CreateRequest` are somewhat different from each other, and I think there's value in a consistent pattern across `File`, `Block`, and `Tag` (and all model objects, really).

Unless I'm reading the code wrong, it looks like you could only use the new Tag subclasses for creation, right?  Could they just subclass Tag.CreateRequest instead?

Reply:
It'd be easier if the two things were split into different PRs, but I agree it'd be lovely to get rid of the `.CreateRequest` objects and don't really mind rolling it up into this one.

Is there a way to get rid of ALL of the `CreateRequest` request objects in this PR so that it's a complete switch over?

Reply:
Sure! Was just testing how the removal of CreateRequest would impact the codebase :) Going to revert these changes and keep that discussion open in another PR